lib/slack.py

```diff
@@ -120,6 +121,13 @@ def search(self, search_query):
 
         return results['videos']
 
+    def verify_token(token):
```

Comment:
This won't compile, all functions inside a class must have `self` as the first argument